config/locales/links/en.yml

```diff
@@ -12,7 +12,7 @@ en:
       auth_app_fallback_html: ' or %{link}.'
       fallback_to_sms_html: Send me a text message with the code instead
       fallback_to_voice_html: If you can't get text message right now, you can get a security code via %{link}
-    privacy_policy: Privacy Policy
+    privacy_policy: Privacy & security
```

Comment:
yes the `&` needs HTML-escaping if we use it. I've seen mixed use on the site, and I personally like the longhand better.